cms/static/cms/js/modules/cms.modal.js

```diff
@@ -50,6 +50,9 @@ var CMS = window.CMS || {};
                 // event emitter
                 this._setupEventEmitter();
 
+                // reference messaging system
+                this.messages = new CMS.Message();
```

Comment:
yeah thought about it too, though handling in separately gives more freedom and is detached from CMS.API